src/inference/hmc.jl

```diff
@@ -674,5 +501,79 @@ function AHMCAdaptor(alg::AdaptiveHamiltonian; init_ϵ=alg.init_ϵ)
     return adaptor
 end
 
-
 AHMCAdaptor(alg::Hamiltonian) = nothing
+
+##########################
+# HMC State Constructors #
+##########################
+
+function HMCState(
```

Comment:
This one for static HMC does not do `link!` and `invlink!` now.

Also can we possible merge them? The only difference is `init_ϵ` in fact. I guess we are safe to rename `init_ϵ` for NUTS to just `ϵ` so two states constructores could be merged.

Reply:
I did this in 2571443